certified-connectors/Kanbanize/apiDefinition.swagger.json

```diff
@@ -0,0 +1,910 @@
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "x-ms-visibility": "advanced",
+            "default": "application/json",
+            "description": "Always use: application/json",
+            "x-ms-summary": "Content Type"
+          },
+          {
+            "name": "Accept",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "x-ms-visibility": "advanced",
+            "default": "application/json",
+            "description": "Always use: application/json",
+            "x-ms-summary": "Content Type"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "card_id": {
+                  "type": "string",
+                  "description": "card_id"
+                },
+                "time": {
+                  "type": "string",
+                  "description": "time"
+                },
+                "comment": {
+                  "type": "string",
+                  "description": "comment"
+                }
+              }
+            }
+          }
+        ],
+        "description": "Log time to a card. The time is logged in minutes."
+      }
+    },
+    "/api/v2/boards": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "data": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "board_id": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "board_id"
+                      },
+                      "workspace_id": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "workspace_id"
+                      },
+                      "is_archived": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "is_archived"
+                      },
+                      "name": {
+                        "type": "string",
+                        "description": "name"
+                      }
+                    }
+                  },
+                  "description": "data"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Boards",
+        "operationId": "GetBoards",
+        "parameters": [],
+        "description": "Get all boards from the account."
+      }
+    },
+    "/index.php/api/kanbanize/create_new_task/format/json": {
+      "post": {
+        "responses": {
+          "default": {
```

Comment:
@dimitar-kanbanize Please replace the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.